src/schema/v2/home/home_page_fairs_module.ts

```diff
@@ -17,22 +19,36 @@ export const HomePageFairsModuleType = new GraphQLObjectType<
           sort: "-start_at",
           active: true,
```

Comment:
Tricky! Looks like the intent here is to show the "freshest" events first, but that gets complicated by the fact that some of these so-called "active" events [haven't actually started yet](https://github.com/artsy/gravity/blob/6b51e8221dd8038f07e184661c4e21d61bd9e431/app/models/util/event_status.rb#L23).

The fix below looks reasonable to me 👍

Reply:
Yeah exactly. Another option we considered was to make two API calls to Gravity, one for `status=current` and another for `status=upcoming`, but it seemed like overkill and we were getting back more data than we expected (so there's something nontrivial about how those APIs handle the `status` param). 

Thanks Roop!